localization/pose_initializer/src/pose_initializer/pose_initializer_node.cpp

```diff
@@ -27,5 +25,4 @@ int main(int argc, char ** argv)
   executor.spin();
   executor.remove_node(node);
   rclcpp::shutdown();
-  return 0;
```

Comment:
Could you tell me why you deleted it?

Reply:
This is a mistake in local work, so I will fix it. I tried to replace main function with macro.

Reply:
updated https://github.com/autowarefoundation/autoware.universe/pull/1500/commits/5ab1c2a7a072f553f44ec6650167d1ff72ba3bf2